### PR TITLE
BAU-294 Permalink print page styling

### PR DESCRIPTION
**Pull request by mrmarvino**
Added 'Print this page' CTA to permalink page. 
Added some extra props to the print button component to allow for better alignment within the permalink page.
Updated print styles to make better use of space for text, and also hidden unnecessary elements on the printed page.

### CODE REVIEW

File: src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx [hunk 1]
```diff
@@ -6,7 +6,17 @@ import {
 } from '@frontend/services/googleAnalyticsService';
 import styles from './PrintThisPage.module.scss';
 
-const PrintThisPage = ({ analytics, ...props }: AnalyticProps) => {
+interface Props {
+  noMargin?: boolean;
+  alignCentre?: boolean;
```
Comment (ntsim) [inline src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx]: A better prop here would just be a single `className`. These props are too specific to the visual styling and will probably have limited usefulness outside of the context of `PermalinkPage`.

File: src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx [hunk 2]
```diff
@@ -18,7 +28,10 @@ const PrintThisPage = ({ analytics, ...props }: AnalyticProps) => {
   return (
     <div
       className={classNames(
-        'govuk-!-margin-top-6',
+        {
+          'govuk-!-margin-top-6': !noMargin,
+          [styles.alignCentre]: alignCentre,
```
Comment (ntsim) [inline src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx]: Is there not a `govuk` util class for `text-align: center`? If not, we should have one under the `dfe` namespace really.

File: src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx (hunk 1 above)
Comment (mrmarvino) [reply src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx]: changed to single className prop

File: src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx (hunk 2 above)
Comment (mrmarvino) [reply src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx]: added in dfe-align classes to common

**Pull request merged**